spec/featureSpec.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+describe("Feature Test", () => {
+  let game;
+
+  beforeEach(() => {
+    game = new Game();
+  });
+
+  it("bowls the Makers Example game", () => {
+    game.bowl(1);
+    game.bowl(4);
+    game.bowl(4);
+    game.bowl(5);
+    game.bowl(6);
+    game.bowl(4);
+    game.bowl(5);
+    game.bowl(5);
+    game.bowl(10);
+    game.bowl(0);
+    game.bowl(1);
+    game.bowl(7);
+    game.bowl(3);
+    game.bowl(6);
+    game.bowl(4);
+    game.bowl(10);
+    game.bowl(2);
+    game.bowl(8);
+    game.bowl(6);
+    game.scorecard();
```

Comment:
yes, I was struggling to think of a way to test an outcome - I think my game class needs to be able to simply return the score, rather than only a stringified scorecard.

Reply:
d'oh - Jasmine `toContain` is my friend here: I can check parts the game.scorecard() string with it (like TOTAL SCORE: 300 for example). Have updated.